src/platform/site-wide/mega-menu/tests/my-va-link/mega-menu.my-va.cypress.spec.js

```diff
@@ -1,4 +1,4 @@
-import manifest from 'applications/mhv/landing-page/manifest.json';
+import manifest from 'applications/personalization/dashboard/manifest.json';
```

Comment:
This test for `/my-va` was importing the MHV Landing Page manifest, instead of the appropriate manifest file.